packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMarkerUserData.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+
+/// Defines user data object for markers.
+@interface GoogleMarkerUserData : NSObject
```

Comment:
`FGMGoogleMarkerUserData`. All classes must be prefixed to avoid runtime collisions.

Reply:
Other files have FLT prefix, I used it, if this really need to use FGM I can rename them again

Reply:
See https://github.com/flutter/flutter/issues/102601 for context on why many files currently use `FLT`.